rpcs3/rpcs3qt/tooltips.h

```diff
@@ -78,6 +78,7 @@ class Tooltips : public QObject
 		const QString set_daz_and_ftz              = tr("Never use this.");
 		const QString accurate_getllar             = tr("Never use this.");
 		const QString accurate_putlluc             = tr("Never use this.");
+		const QString accurate_rsx_access          = tr("Never use this.");
```

Comment:
Actually you make this "gamefix" becouse you had a reason. So give it reason egz: 
"Protect SPU reservation from the evil RSX which may damage it.
Fixes God Of War Ascension with non-tsx when enabled."
Not generic "Never use this" because:
1) If somone wants to remove this he needs to know why it was here in first place
2) If it isn't supposed to be used then it don't have any sense to even program this let alone merge.

I hope you take my suggestion into account before merging this.

Reply:
I will commit a description for this setting and the others.. in later pull requests.
It's not urgent, this pr contains mostly the technical parts of the setting.

Reply:
1. There's git history. No one is going to remove whole features without knowing why they were there, specifically removing accurate emulation out of nowhere.
2. If it wasn't supposed to be used it wouldn't be committed. Needs an accurate description of what it does, but it isn't a gamefix, just an accurate setting that heavily impacts performance and can't be on by default.